docs/developer_guide.md

```diff
@@ -18,15 +18,21 @@ cargo install --force --version 0.18.0 cbindgen
 cargo install --force --version 0.57.0 bindgen
 ```
 
-## Tor tests
+The versions of bindgen and cbindgen you install should match the [versions installed in the CI](https://github.com/shadow/shadow/blob/dev/.github/workflows/lint.yml).
 
-Shadow includes tests that use Tor and TGen. These are not run by default. To run them, first make sure that both tor and tgen are located at `~/.shadow/bin/{tor,tgen}`. These can be symlinks to tor and tgen binaries elsewhere in the filesystem. It is recommended to build Shadow in release mode, otherwise the tests may take much longer to complete.
+## Extra tests
+
+Shadow includes tests that require additional dependencies (for example Tor, TGen, and networkx) and are not run by default, but are run as part of the CI tests. To run them locally, first make sure that both tor and tgen are located at `~/.shadow/bin/{tor,tgen}`. These can be symlinks to tor and tgen binaries elsewhere in the filesystem. You should also install all of Shadow's optional dependencies.
```

Comment:
First sentence is a little awkward. Maybe break into two: "Shadow includes tests that require additional dependencies, such as Tor, TGen, and networkx. These aren't run by default, but are run as part of the CI tests."

Reply:
Fixed.